.gitignore

```diff
@@ -8,3 +8,4 @@
 composer.lock
 vendor/
 app/config/config.dev.php
+/nbproject/
```

Comment:
Please do not use this ignore file to define platform specific files.

For these purposes create a global `.gitignore` file, which is a list of rules for ignoring files in every Git repository on your computer.

See: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore